.github/workflows/docker.yml

```diff
@@ -20,9 +20,26 @@ jobs:
         uses: actions/checkout@v2
         with:
           java-version: 1.8
+          fetch-depth: 0
+      -
+        name: Setup GitVersion action
+        uses: gittools/actions/gitversion/setup@v0.9.7
+        with:
+          versionSpec: '5.x'
+      -
+        name: Execute GitVersion action
+        id: gitversion
+        uses: gittools/actions/gitversion/execute@v0.9.7
+      -
+        name: Publish release on GitHub
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: '${{ steps.gitversion.outputs.semver }}'
       -
         name: Grant execute permission for gradlew
         run: chmod +x gradlew
       -
         name: Build with Gradle and Push
-        run: ./gradlew jib
+        run: |
+          ./gradlew jib \
+          -Djib.to.tags='latest',$GITVERSION_SEMVER
```

Comment:
what would be `GITVERSION_SEMVER`?

Reply:
It is an output from gitversion action. When gitverson is executed it provides a number of outputs (see [here](https://github.com/GitTools/actions/blob/main/docs/examples/github/gitversion/execute/usage-examples.md) for full list) and semVer is one of them. It will contain the calculated version for a build (see [here](https://github.com/GitTools/actions/blob/main/docs/examples/github/gitversion/execute/usage-examples.md#example-6) for example)

Reply:
ok got it. can you please add few sample versions here which will be the output of GITVERSION_SEMVER?

Reply:
Sample outputs of GITVERSION_SEMVER on DockerHub :
<img width="103" alt="image" src="https://user-images.githubusercontent.com/97123508/184517424-059b98c5-a672-4596-b9a3-72da880fbfc4.png">
<img width="205" alt="image" src="https://user-images.githubusercontent.com/97123508/184517486-10ddd6df-0721-4ab4-8353-c384b9a78c9c.png">
<img width="191" alt="image" src="https://user-images.githubusercontent.com/97123508/184517494-7c247dcd-a2f3-4f31-94d0-f7d99d81dd5e.png">

Reply:
so we already have 10 tags ... will this auto increment start from those tags. i.e. we have 1.10.0 tag so it will start as 1.10.1 or 1.11.0 or 2.0.0 right ?